src/main/java/uk/gov/hmcts/reform/sscs/services/ccd/UpdateCcdRpc.java

```diff
@@ -18,7 +18,9 @@ boolean updateCcdRpc(SscsCaseData gapsCaseData, SscsCaseData existingCcdCaseData
 
         RegionalProcessingCenter gapsRpc = gapsCaseData.getRegionalProcessingCenter();
         RegionalProcessingCenter existingRpc = existingCcdCaseData.getRegionalProcessingCenter();
-        if (hasRpcChanged(gapsRpc, existingRpc)) {
+
+        if (hasDwpRpcNameChanged(gapsCaseData.getDwpRegionalCentre(), existingCcdCaseData.getDwpRegionalCentre())
```

Comment:
is Dwp Regional centre not depend on Regional Processing center? I think we should have one check as if regional processing center change then update all relevant info and dwp regional center